code/modules/admin/admin_verbs.dm

```diff
@@ -806,7 +806,6 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 			continue
 		GM.parse_gas_string(F.initial_gas_mix)
 		F.copy_air(GM)
-		F.update_visuals()
```

Comment:
Missing `singulostation edit` marker
Also, do you happen to know why this is removed?

Reply:
Not sure, it was removed in your PR
https://github.com/SinguloStation13/SinguloStation-Blue/pull/2/files#diff-46efb60da866bae636d30b19603fa3a90545244fd0f1f5b154edaf1e11b41daf